index.html

```diff
@@ -668,17 +668,15 @@ <h2>Input toggle</h2>
         <div class="row">
             <div class="column">
                 <div class="flex items-center">
-                    <input class="mi-toggle" id="toggle-1" type="checkbox" />
-                    <label class="mi-toggle-btn" for="toggle-1"></label>
+                    <input type="checkbox" id="toggle-1" class="mi-toggle" />
                     <label for="toggle-1" class="mi-label">Toggle</label>
                 </div>
             </div>
 
             <div class="column">
                 <div class="flex items-center">
-                    <input class="mi-toggle" id="toggle-2" disabled type="checkbox" />
-                    <label class="mi-toggle-btn" for="toggle-2"></label>
-                    <label class="mi-label">Toggle disabled</label>
+                    <input type="checkbox" id="toggle-2" class="mi-toggle" disabled />
+                    <label for="toggle-2" class="mi-label">Toggle disabled</label>
```

Comment:
Is there a reason why we can't have the input inside the label?

Reply:
No limitations regarding that. Wrapping and not using the `id` and `for` attributes is also possible the only reason that I'm not doing it, in this case, is to make an isolated presentation of how the toggle can be used.